app/services/idv/steps/in_person/verify_step.rb

```diff
@@ -33,6 +34,10 @@ def add_proofing_component
             update(document_check: Idp::Constants::Vendors::USPS)
         end
 
+        def capture_secondary_id_enabled
```

Comment:
Since this method returns a boolean, I might expect it to be conventionally suffixed with a `?`

```suggestion
        def capture_secondary_id_enabled?
```